src/providers/krb5/krb5_child.c

```diff
@@ -1370,7 +1371,11 @@ static krb5_error_code sss_krb5_prompter(krb5_context context, void *data,
                                     prompts[c].prompt);
         }
 
-        DEBUG(SSSDBG_FUNC_DATA, "Prompter interface isn't used for password prompts by SSSD.\n");
+        err_msg = krb5_get_error_message(context, KRB5_LIBOS_CANTREADPWD);
+        DEBUG(SSSDBG_FUNC_DATA,
+              "Prompter interface isn't used for prompting by SSSD."
+              "Error [%ld/%s] is expected.\n", KRB5_LIBOS_CANTREADPWD, err_msg);
```

Comment:
If I read this message in the logs, I understand that some function returned this error which is expected. But in reality, this function is returning this error.
May I propose this?
```
       DEBUG(SSSDBG_FUNC_DATA,
              "Prompter interface isn't used for prompting by SSSD."
              "Returning the expected error [%ld/%s].\n", KRB5_LIBOS_CANTREADPWD, err_msg);
```

Reply:
fixed

Reply:
Thank you.